datadotworld/files.py

```diff
@@ -60,18 +63,20 @@ def __init__(self, config, dataset_key, file_name,
             call by default will wait indefinitely for a response)
         """
         self._api_host = "https://api.data.world/v0"
+        self._query_host = "https://query.data.world"
```

Comment:
question for @rflprr and @tle  - I think you have also implemented these download operations under api.data.world - is that live?  if so, can consolidate this to a single host